### PR TITLE
🐛Source Freshdesk: Update parsing of start_date param

### LINKED ISSUE #13328: Freshdesk connection: `It should be a valid date in the 'combined date and time ISO8601' format`
## Environment
- **Airbyte version**:  0.39.5-alpha
- **OS Version / Instance**: AWS EKS
- **Deployment**: k8s
- **Source Connector and version**: Freshdesk version 0.3.0
- **Destination Connector and version**: s3 0.3.6
- **Severity**:  Critical
- **Step where error happened**: Sync job

## Current Behavior
Syncing Freshdesk to s3 (as JSON, gzip compression) fails:

## Expected Behavior
No error.
## Logs
Screenshot 2022-05-31 at 13 30 36
Full logs:
[airbyte-freshdesk-logs.txt](https://github.com/airbytehq/airbyte/files/8805913/airbyte-freshdesk-logs.txt)
## Steps to Reproduce
1. set up s3 destination (JSON, gzip)
2. set up Freshdesk source (with domain, API, no date limit, no rate limit)
3. create connection, sync all (full refresh-append, mirror source structure)

## Are you willing to submit a PR?
Maybe if it is simple.

### PR DESCRIPTION
**Pull request by annalvova05**
## What
Sync failed if `start_date` was not specified in config. Param for incremental sync is passed not in the correct format.

## How
Update format of date-time for Incremental Sync.

## Recommended reading order
1. `streams.py`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Review by lazebnyi — APPROVED** [review]

File: airbyte-integrations/connectors/source-freshdesk/source_freshdesk/streams.py [hunk 1]
```diff
@@ -34,9 +34,7 @@ def __init__(self, authenticator: AuthBase, config: Mapping[str, Any], *args, **
         # Since this logic rely not on updated tickets, it can break tickets dependant streams - conversations.
         # So updated_since parameter will be always used in tickets streams. And start_date will be used too
         # with default value 30 days look back.
-        self.start_date = (
-            pendulum.parse(config.get("start_date")) if config.get("start_date") else pendulum.now() - pendulum.duration(days=30)
-        )
+        self.start_date = config.get("start_date") or (pendulum.now() - pendulum.duration(days=30)).strftime("%Y-%m-%dT%H:%M:%SZ")
```
Comment (midavadim) [inline airbyte-integrations/connectors/source-freshdesk/source_freshdesk/streams.py:37]: start_date was a datetime before, now it is a string. 

Is it expected further in the code ?

Comment (annalvova05) [reply airbyte-integrations/connectors/source-freshdesk/source_freshdesk/streams.py:37]: It's expected

File: airbyte-integrations/connectors/source-freshdesk/unit_tests/test_300_page.py [hunk 2]
```diff
@@ -11,70 +11,70 @@
 def responses_fixtures():
     return [
         {
-            "url": "/api/tickets?per_page=1&updated_since=2002-02-10T22%3A21%3A44%2B00%3A00",
```
[the remark this answers is not in the document]
Comment (annalvova05) [reply airbyte-integrations/connectors/source-freshdesk/unit_tests/test_300_page.py:14]: Freshdesk expects ISO8601 `date-time` format, but for some reason, the timezone isn't supported. 
Freshdesk returns `date-time` value in formats like `"YYYY-MM-DDTHH:mm:SSZ"` and expects in filtering the same format or without `Z` at the end. 
I think maybe filtering was not tested rarely or Freshdesk has a bug in the `date-time` format.

**Review by midavadim — APPROVED** [review]

**Pull request merged**